.github/workflows/reusable-docker-build-push.yml

```diff
@@ -269,8 +351,12 @@ jobs:
           secrets: ${{ secrets.DOCKER_SECRETS }}
 
           # https://docs.docker.com/build/ci/github-actions/examples/#cache
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+          # https://aws.amazon.com/blogs/containers/announcing-remote-cache-support-in-amazon-ecr-for-buildkit-clients/
+          # Embeds the build cache into a separate image
+          cache-from: |
+            type=registry,ref=${{ env.ECR_REGISTRY }}:cache
```

Comment:
Possible bug: env.ECR_REGISTRY might not be set (ref [line 237](https://github.com/oslokommune/reusable-docker-build-push/blob/68bad547e55baa83e2cbd7d5c67016b0938ea78d/.github/workflows/reusable-docker-build-push.yml#L237)).

Reply:
Discussion in pair programming: We'll remove this from this PR, as it is unrelated to the rest of the changes.